### PR TITLE
Cli/faq] 部署環境 / 雲端物件儲存服務

### PR DESCRIPTION
**Pull request by ting88510**
#268

### CODE REVIEW

File: CLI/4. FAQs/1. TWCC CLI - Install.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+{%hackmd @docsharedstyle/default %}
+
+# TWCC FAQs | TWCC CLI: Deployment environment
+
+:::spoiler Q1. How to install TWCC-CLI?
+
+:::info
+1. Use any Python package manager with the `$ pip install -U TWCC-CLI` command. Or, see [<ins>TWCC-CLI v0.5</ins>](https://github.com/TW-NCHC/TWCC-CLI/tree/v0.5) operation manual for installation and use.
```
Comment (VigaWei) [inline CLI/4. FAQs/1. TWCC CLI - Install.md:14]: use. > usage.

File: CLI/4. FAQs/2. TWCC CLI - COS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+{%hackmd @docsharedstyle/default %}
+
+# TWCC FAQs | TWCC CLI: Cloud Object Storage (COS)
+
+:::spoiler Q1. Why do I fail when uploading and downloading a large number of files using TWCC-CLI?
+
+:::info
+Currently TWCC CLI file management function is still developing and optimizing. If you need to transfer a large number of files, see [<ins>this document</ins>](https://man.twcc.ai/@twccdocs/doc-cos-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Fcosbackup-en) to use s3cmd commands to operate.
```
Comment (VigaWei) [inline CLI/4. FAQs/2. TWCC CLI - COS.md:14]: Currently TWCC CLI file management function is still developing and optimizing. > 

Currently, the TWCC CLI file management function is still developing and optimizing.

Comment (VigaWei) [reply CLI/4. FAQs/2. TWCC CLI - COS.md:14]: If you need to transfer a large number of files, see [<ins>this document</ins>](https://man.twcc.ai/@twccdocs/doc-cos-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Fcosbackup-en) to use s3cmd commands to operate. >

To transfer a large number of files, refer to  [<ins>this document</ins>](https://man.twcc.ai/@twccdocs/doc-cos-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Fcosbackup-en)  to operate with s3cmd.

**Review by VigaWei — COMMENTED** [review]
Thx! Please revise the doc according to the comments.

**Comment by ting88510** [comment]
Revised, thanks!!

**Pull request merged**